src/content/en/2021/caching.md

```diff
@@ -17,4 +20,435 @@ featured_stat_3: TODO
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>CSS</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Image</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Script</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Font</td>
+        <td class="numeric">365</td>
+        <td class="numeric">365</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>TODO: Caption the table above</figcaption>
+</figure>
+
+When we look at the entirety of cacheable and non-cacheable resources,
+
+{{ figure_markup(
+  image="cacheable-responses.png",
+  caption="Percent of cacheable vs non-cacheable responses.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=674929574&format=interactive",
+  sheets_gid="812850996",
+  sql_file="ttl.sql"
+) }}
+
+## How do cache TTLs compare to resource age?
+We see that Images and Videos maintained the same average age whether from 1st or 3rd party resources. Images consistently had the same resource age of 2 years, while most Video resources were between 8-52 weeks old.
+
+Breaking down the other types of content, we discovered Fonts for 3rd parties are cached the most between 8-52 weeks at 72.4%. However, for 1st party the largest resource age groups is evenly split between 8-52 weeks and over 2 years- quite a large variance.
+We see similar results for Audio and Scripts where the majority of 1st party are between 8-52 weeks old while for 3rd party they are between 1-7 weeks. 
+
+Audio was the most highly cached resource across both 1st and 3rd parties. However, the resource age varied greatly between first party (averaging 8-52 weeks) and third party, at only 1-7 weeks. Audio resources in first party situations tend to be updated less frequently (why?), so third parties may be capitalizing on a caching opportunity by offering fresher resources. 
+
+{{ figure_markup(
+  image="first-party-resource-age-by-content-type.png",
+  caption="Distribution of first-party resource age by content type (mobile only).",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=597168369&format=interactive",
+  sheets_gid="856268091",
+  sql_file="resource_age_party_and_type_wise_groups.sql"
+) }}
+
+The largest group of cached 1st party CSS (32.2%) tended to be 8-52 weeks old, while the largest group for 3rd parties was less than a week with 51.8% of resources cached for that duration.
+
+Finally, HTML has the largest 1st party group served with less than a week with 42.7% and third party's largest group is between 1-7 weeks with 43.1%. 
```

Comment:
Edit: 

Finally, HTML has the largest 1st party group served with less than a week with 42.7% and 3rd party's largest group is between 1-7 weeks with 43.1%.